spec/requests/new_document_spec.rb

```diff
@@ -4,9 +4,10 @@
   describe "GET /documents/choose-document-type" do
     it "shows supertype when a supertype managed by content publisher is selected" do
       supertype = Supertype.all.reject(&:managed_elsewhere).first
-      get choose_document_type_path, params: { supertype: supertype.id }
+      supertype_id = supertype.id
```

Comment:
Minor: the character difference between `supertype_id` and `supertype.id` suggests this variable isn't necessary.

Reply:
good spot, not completely sure why I added this